tests/test_htmIndex.py

```diff
@@ -70,26 +72,35 @@ def make_sky_catalog(out_path, size=1000):
         is_variable = np.random.randint(2, size=size)
         extra_col1 = np.random.normal(size=size)
         extra_col2 = np.random.normal(1000., 100., size=size)
+        pm_ra = -np.ones(size)*cls.proper_motion  # arcsec/year
+        pm_dec = np.zeros(size)  # arcsec/year
```

Comment:
Worth a comment here to make it clear that the intention is for the proper motion to be explicitly in ra only.

Reply:
Better yet I'll put the PM at an angle with `cls.proper_motion_amt` and `cls.proper_motion_dir` and make them instances of `lsst.geom.Angle`